cms/static/js/views/move_xblock_list.js

```diff
@@ -104,10 +104,15 @@ function($, Backbone, _, gettext, HtmlUtils, StringUtils, XBlockUtils, MoveXBloc
                 this.childs_info.childs = [];
             }
 
+            this.validateMoveOperation();
             this.setDisplayedXBlocksCategories();
             this.render();
         },
 
+        validateMoveOperation: function() {
+            Backbone.trigger('move:validateMoveOperation', this.parent_info.parent);
+        },
```

Comment:
The name is confusing, can you rename it to `enableDisableMoveButton` or `triggerEnableDisableMoveOperation`